impeller/entity/entity.cc

```diff
@@ -174,6 +174,11 @@ Capture& Entity::GetCapture() const {
   return capture_;
 }
 
+Entity Entity::Clone() const {
+  Entity copy = *this;
+  return copy;
```

Comment:
`return Entity(*this);`?  I want to make to make the decision to elide the copy as easy as possible for the compiler.

Reply:
Done